zenoh/src/net/routing/dispatcher/queries.rs

```diff
@@ -186,7 +186,7 @@ pub(crate) fn undeclare_queryable(
         Resource::clean(&mut res);
         drop(wtables);
     } else {
-        tracing::error!("{} Undeclare unknown queryable {}", face, id);
+        tracing::trace!("{} Undeclare unknown queryable {}", face, id);
```

Comment:
Should we set to debug rather than trace ?

Reply:
I think it should be left at Trace for the same reason I gave for comment on `pubsub`

Reply:
Fine. 
Maybe add a comment here stating that this is expected in case of ACL.